benefits/static/css/styles.css

```diff
@@ -65,11 +65,15 @@ footer {
   padding-bottom: 1rem;
 }
 
+footer a {
```

Comment:
For the sake of specificity, could you target this style for `footer .footer-links a`? That way, in case we ever add different links to the footer, outside of this `<ul>`, we won't have to override the `a` element entirely.